pom.xml

```diff
@@ -20,19 +20,32 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>nl.knaw.dans.shared</groupId>
-        <artifactId>dans-prototype</artifactId>
+        <artifactId>dans-scala-prototype</artifactId>
         <version>1.32</version>
     </parent>
     <groupId>nl.knaw.dans.easy</groupId>
     <artifactId>easy-schema</artifactId>
     <name>EASY Schema Definitions</name>
     <description>Collection of metadata schema's used by EASY</description>
     <version>1.x-SNAPSHOT</version>
-    <inceptionYear>2015-2016</inceptionYear>
+    <inceptionYear>2015-2017</inceptionYear>
```

Comment:
I don't think it is a good idea all sources should have the same inception year, but that is beyond the scope of the fixed issue.